### ISSUE TITLE
[FEAT] Colorized Atlas

### ISSUE THREAD

**Issue by mmsheeks** [Hilistia/smp#1]
**Is your feature request related to a problem? Please describe.**
Cham was describing how the atlas map interface is hard to see due to low contrast and lack of color. This presents as an accessibility issue for the server as a whole, so it needs to be resolved.

**Describe the solution you'd like**
Update the core-mod resource pack to include colorized versions of all sprites used by the atlas. This would also allow us to fix any gaps in implementation as needed, since we'd have existing infrastructure for adding content to the book.

**Describe alternatives you've considered**
If this lift is too much, we can also make a published version of the modpack with accessible map mods such as Journeymap or Xaeros. If we do this, we'll need to ensure the relevant settings don't allow for unfair play balance.

**Comment by mmsheeks** [comment]
see https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1292324-antique-atlas for reference

**Comment by mmsheeks** [comment]
Default Antique Atlas resources
[antique_atlas.zip](https://github.com/user-attachments/files/18138679/antique_atlas.zip)
